tests/test_config.py

```diff
@@ -1887,6 +1887,35 @@ def test_multiple_homedir_relative_local_indexservers(self, newconfig):
         assert config.indexserver['default'].url == expected
         assert config.indexserver['local1'].url == config.indexserver['default'].url
 
+    def test_replace_pathsep_unix(self, monkeypatch, newconfig):
```

Comment:
This could be parametrized:

```python
@pytest.mark.parametrize('pathsep', [':', ';'])
def test_replace_path_sep(self, monkeypatch, newconfig, pathsep)
```